applications/sentence-transformers-ft/modal.py

```diff
@@ -0,0 +1,60 @@
+from datasets import load_dataset
+from sentence_transformers import SentenceTransformer
+import modal
+import pathlib
+from finetune import finetune
+
+
+# Modal constants
+VOL_MOUNT_PATH = pathlib.Path("/vol")
+GPU_CONFIG = "a10g"
+
+
+# Functions for Modal Image build steps (cache the model and dataset)
+# NOTE: this can be removed for simplicity, this is currently here for faster devloop times when repeatedly running the function
+def download_model():
+    model_id = "BAAI/bge-small-en-v1.5"
```

Comment:
should be a script variable

Reply:
Changed to make it a script variable https://github.com/567-labs/fastllm/pull/28/files#diff-7a2ff54b496ec93dabf2bf46fff30a5afbecca3544de44e4f30dc21ed4c84459R12